app/javascript/article-form/components/ArticleCoverImage.jsx

```diff
@@ -46,6 +46,9 @@ const StandardImageUpload = ({
           className="screen-reader-only"
           data-max-file-size-mb="25"
         />
+        <span data-testid="tooltip" className="crayons-tooltip__content">
+          Use a ratio of 100:42 for best results.
+        </span>
```

Comment:
[nitpick, non-blocking]

Not in love with this copy. All suggestions appreciated.